workshops/20_python_beginners_workshop/tasks/short_tasks/practice_boolean.md

```diff
@@ -0,0 +1,27 @@
+# Exercise 2: Boolean & Comparisons
+
+This is the way how you usually write comments in Python:
+
+~~~Python
+# Hi! I'm Anna and I live in practice_boolean.md.
```

Comment:
that is a weird phrase !

Reply:
I changed it to "Hi! I'm Anna and I am living in coding.py." and later on I explain that this is just a joke ;) Will they understand?